### ISSUE TITLE
[1.9-3.0.6] Block spam if looking at basic rf storage while holding electric pickaxe.

### ISSUE THREAD

**Issue by DanWehr** [TeamBR-Modding/NeoTech#311]
What Version of NeoTech are you using: 1.9-3.0.6, single player world.

If possible, did you break and replace the block: Yes, I tried breaking the storage block and replacing, and bug persisted.

Nothing happens if I'm close to a Basic RF Storage block and look at it holding anything else, but if I'm holding an electric pickaxe, copies of Basic RF Storage blocks start flying in every direction, at a rate of what appears to be one per tick. The end result is this:

http://imgur.com/a/ySyQE

These are not actually proper duplicates. They can not be picked up, and are gone when leaving and reentering the game.

Placement of the Basic RF Storage block doesn't matter (connected to other pipes/machines, or on its own). The charge level of the pickaxe also doesn't matter. It's empty in the pic, but the same thing happens with it fully charged.

**Comment by dreness** [comment]
+1, although in my case it was an Elite storage block. Same version. I'm using the pick from Psi for now as a workaround.

**Comment by ekthomson** [comment]
I've encountered this as well. I created a new electric pickaxe to try it again, and discovered that an un-upgraded electric pickaxe does not do this. I added modules one at a time until I found it happening again. The last module I added was an Area of Effect mod, and that is when it started triggering the spammed particles. Did you both have this mod as well? 

Also, it happens with all 3 storage blocks, as well as the redstone power block from Modular Systems.
